### DISCUSSION TITLE
Pocket Query - My finds

### DISCUSSION THREAD

**Discussion by mcmellow** [2Abendsegler/GClh#2063] [Questions] [answered]
GC Little Helper II version 0.11.12
Chrome Versie 98.0.4758.102 (Officiële build) (64-bits)

Pressing button "Add to Queue to run the My Finds Pocket Query does not work any more. Pressing this button nothing happen. 
Switching off  GC Little Helper in TamperMonkey makes it possible to start the My Finds pocket query. After that I switch on GC little Helper again in TamperMonkey.

Is this a bug or a setting somewhere.

**Comment by 2Abendsegler** [comment]
Hi,

Thanks for reporting the bug.
We have already opened issues for that bug.

Cheers
Frank

**Comment by mcmellow** [comment]
Put up a question, cause I did not find an issue on it. 
Wait for the solution, use the switch off / switch on work around for the time being.

**Comment by 2Abendsegler** [comment, marked as answer]
Moin,
the main issue is #2028. In the meantime there is another issue #2064.
Until we fix the bug, you'll have to work without GClh at this point.

**Comment by mcmellow** [comment]
Thanks for the support. the button **"Add to Queue"**  is working again with GCLH II active.
